### PR TITLE
add lowestaudio / lowestvideo

### PR DESCRIPTION
**Pull request by TimeForANinja**
respond to https://github.com/fent/node-ytdl-core/issues/395

### CODE REVIEW

**Comment by TimeForANinja** [comment]
:thinking:
don't understand why the commit build failed in repo

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ ytdl('http://www.youtube.com/watch?v=A02s8omM_hI')
 
 Attempts to download a video from the given url. Returns a [readable stream](https://nodejs.org/api/stream.html#stream_class_stream_readable). `options` can have the following keys
 
-* `quality` - Video quality to download. Can be an [itag value](http://en.wikipedia.org/wiki/YouTube#Quality_and_formats), a list of itag values, or `highest`/`lowest`/`highestaudio`/`highestvideo`. `highestaudio`/`highestvideo` both prefer audio/video only respectively. Defaults to `highest`.
+* `quality` - Video quality to download. Can be an [itag value](http://en.wikipedia.org/wiki/YouTube#Quality_and_formats), a list of itag values, or `highest`/`lowest`/`highestaudio`/`lowestaudio`/`highestvideo`/`lowestvideo`. `highestaudio`/`lowestaudio`/`highestvideo`/`lowestvideo` both prefer audio/video only respectively. Defaults to `highest`.
```
Comment (fent) [inline README.md]: suggest to change 'both' to 'all'
```suggestion
* `quality` - Video quality to download. Can be an [itag value](http://en.wikipedia.org/wiki/YouTube#Quality_and_formats), a list of itag values, or `highest`/`lowest`/`highestaudio`/`lowestaudio`/`highestvideo`/`lowestvideo`. `highestaudio`/`lowestaudio`/`highestvideo`/`lowestvideo` all prefer audio/video only respectively. Defaults to `highest`.
```

**Comment by fent** [comment]
> 🤔
> don't understand why the commit build failed in repo

It's from a different commit not this one. And it's from the irl-test.js, which sometimes fail. :(

File: README.md (hunk 1 above)
Comment (TimeForANinja) [reply README.md]: sounds plausible

**Comment by fent** [comment]
Thank you!

**Pull request merged**